airflow-provider-fivetran/operators/fivetran.py

```diff
@@ -29,8 +30,11 @@ class FivetranOperator(BaseOperator):
     @apply_defaults
     def __init__(
         self,
-        api_key=None,
-        api_secret=None,
+        run_name: Optional[str] = None,
+        timeout_seconds: Optional[int] = None,
+        fivetran_conn_id: str = 'fivetran',
+        fivetran_retry_limit: int = 3,
+        fivetran_retry_delay: int = 1,
         connector_id=None,
         poll_status_every_n_seconds: int = 15,
```

Comment:
There is probably a better name for this, like `poll_interval` or `poll_interval_seconds`